src/rubintv/static/stylesheets/main.css.map

```diff
@@ -1 +1 @@
-{"version":3,"sourceRoot":"","sources":["main.sass","calendar.sass"],"names":[],"mappings":";AAAQ;ACAR;EACE;EACA;EACA;;;AAEF;EACE;;;AAEF;EACE;EACA;EACA;EACA;EACA;;AACA;EACE;EACA;EACA;EACA;EACA;EACA;;;AAGF;EACE;;AACF;EACE;;;AAGJ;EACE;;;AAEF;EACE;EACA;EACA;EACA;;;AAEF;EACE;EACA;EACA;;;AAEF;EACE;EACA;EACA;EACA;EACA;EACA;EACA;;;AAEF;EACE;EACA;EACA;EACA;;AACA;EACE;;AACF;EACE;EACA;;AACF;EACE;EACA;;;AAEJ;EACE;;;ADhEF;EACC;EACA;EACA;;;AAED;EACC;EACA;;;AAED;EACC;EACA;EACA;EACA;;;AAED;EACC;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;;;AAED;EACC;EACA;EACA;;;AAED;EACC;EACA;;;AAED;EACC;EACA;EACA;;;AAED;EACC;EACA;;AACA;EACC;;;AAEF;EACC;;;AAED;EACC;;;AACD;EACC;;;AACD;EACC;;;AACD;EACC;;;AACD;EACC;EACA;;;AAED;EACC;EACA;EACA;;;AAED;EACC;;;AAED;EACC;;;AAED;EACC;EACA;EACA;;AAEA;EACC;EACA;;AAED;EACC;EACA;;AACA;EACC;;;AAEH;EACC;;;AAED;EACC;;AACA;EACC;EACA;;;AAEF;EACC;EACA;;;AAED;EACC;EACA;;AACA;EACC;;;AAEF;EACC;;;AAED;EACC;EACA;EACA;;;AAED;EACC;;;AAED;EACC;;AACA;EACC;EACA;EACA;;;AAEF;EAEE;IACC;IACA;IACA;IACA;;;AAGH;EACC;;;AAED;EACC;EACA;EACA;;;AAED;EACC;;AACA;EACC;EACA;EACA;EACA;;;AAEF;EACC;EACA;EACA;EACA;;;AAED;EACC;;AACA;EACC;;;AAEF;EACC;EACA;EACA;;;AAGA;EACC;EACA;;;AAEF;EACC;EACA;EACA;EACA;EACA;EACA;;AACA;EACC;EACA;EACA","file":"main.css"}
\ No newline at end of file
+{"version":3,"sourceRoot":"","sources":["main.sass","calendar.sass"],"names":[],"mappings":";AAAQ;ACAR;EACE;EACA;EACA;EACA;;;AAEF;EACE;;;AAEF;EACE;EACA;EACA;EACA;EACA;;AACA;EACE;EACA;EACA;EACA;EACA;EACA;;;AAGF;EACE;;AACF;EACE;;;AAGJ;EACE;;;AAEF;EACE;EACA;EACA;EACA;;;AAEF;EACE;EACA;EACA;;;AAEF;EACE;EACA;EACA;EACA;EACA;EACA;EACA;;;AAEF;EACE;EACA;EACA;EACA;;AACA;EACE;;AACF;EACE;EACA;;AACF;EACE;EACA;;;AAEJ;EACE;;;ADjEF;EACC;EACA;EACA;;;AAED;EACC;EACA;;;AAED;EACC;EACA;EACA;EACA;;;AAED;EACC;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;;;AAED;EACC;EACA;EACA;;;AAED;EACC;EACA;;;AAED;EACC;EACA;EACA;;;AAED;EACC;EACA;;AACA;EACC;;;AAEF;EACC;;;AAED;EACC;;;AACD;EACC;;;AACD;EACC;;;AACD;EACC;;;AACD;EACC;EACA;;;AAED;EACC;EACA;EACA;;;AAED;EACC;;;AAED;EACC;;;AAED;EACC;EACA;EACA;;AAEA;EACC;EACA;;AAED;EACC;EACA;EACA;;AAEA;EACC;;AACA;EACC;EACA;;AAEF;EACC;EACA;EACA;EACA;EACA;EACA;EACA;EACA;;AACA;EACC;EACA;EACA;;;AAGJ;EACC;;;AAED;EACC;EACA;;AACA;EACC;EACA;;;AAEF;EACC;EACA;;;AAED;EACC;EACA;;AACA;EACC;;;AAEF;EACC;;;AAED;EACC;;;AAED;EACC;EACA;EACA;;;AAED;EACC;;;AAED;EACC;;AACA;EACC;EACA;EACA;;;AAEF;EAEE;IACC;IACA;IACA;IACA;;;AAGH;EACC;;;AAED;EACC;EACA;EACA;;;AAED;EACC;;AACA;EACC;EACA;EACA;EACA;;;AAEF;EACC;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;;AACA;EACC;;;AAEF;EACC;;AACA;EACC;;;AAEF;EACC;EACA;EACA;;;AAED;EACC;EACA;EACA;;AACA;EACC;EAGA;EACA;EACA;;;AAEF;EACC;EACA;EACA;;AACA;EACC;EACA;EACA;;;AAEF;EACC;EACA;EACA;EACA;;;AAED;EACC;EACA;EACA;EACA;;;AAED;EACC;EACA;EACA;EACA;EACA;EACA;EACA;;;AAED;EACC;EACA;EACA;EACA;EACA;EACA;EACA;;AACA;EACC;EACA;EACA;EACA;;AACA;EACC;;;AAGF;EACC;;AACD;EACC;;;AAEF;EACC;EACA;;;AAED;EACC;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;;AACA;EACC;EACA;EACA;EACA;;AACD;EACC","file":"main.css"}
```

Comment:
github complains about the missing newline at the end-of-file.

Reply:
This is an auto-generated file